deploy/consts.ts

```diff
@@ -80,6 +80,10 @@ export const OP_STACK_ADDRESS_MAP: {
       L1CrossDomainMessenger: "0x5D4472f31Bd9385709ec61305AFc749F0fA8e9d0",
       L1StandardBridge: "0x697402166Fbf2F22E970df8a6486Ef171dbfc524",
     },
+    [CHAIN_IDs.INK]: {
+      L1CrossDomainMessenger: "0x69d3cf86b2bf1a9e99875b7e2d9b6a84426c171f",
+      L1StandardBridge: "0x88ff1e5b602916615391f55854588efcbb7663f0",
```

Comment:
If the OP adapter deployment script is used then we'd want to also define `L1OpUSDCBridgeAdapter` as 0x0 here.

Reply:
https://github.com/across-protocol/contracts/pull/808/commits/8a97edf2a2f96c7527eb8993ec18018443b4015b